### PR TITLE
FUSETOOLS2-1746: Provide UI test for Camel Debugger launch configurat…

### PR DESCRIPTION
**Pull request by pospisilf**
…ion snippet using JBang

### CODE REVIEW

**Comment by pospisilf** [comment]
blocked by https://issues.redhat.com/browse/FUSETOOLS2-2264

**Review by djelinek — CHANGES_REQUESTED** [review]

File: src/ui-test/tests/command.palette.test.ts [hunk 1]
```diff
@@ -25,15 +25,15 @@ import {
...
+    CAMEL_RUN_ACTION_QUICKPICKS_LABEL,
+    CAMEL_RUN_DEBUG_ACTION_QUICKPICKS_LABEL,
     waitUntilTerminalHasText,
     TEST_ARRAY_RUN,
     TEST_ARRAY_RUN_DEBUG,
     executeCommand,
     disconnectDebugger,
     killTerminal,
-    CAMEL_ROUTE_YAML_WITH_SPACE,
+    CAMEL_ROUTE_YAML_WITH_SPACE
```
Comment (djelinek) [inline src/ui-test/tests/command.palette.test.ts:36]: weird spaces at the end of this line

File: src/ui-test/utils.ts [hunk 2]
```diff
@@ -395,3 +397,43 @@ export async function waitUntilEditorIsOpened(driver: WebDriver, title: string,
...
+export async function createFolder(folder: string): Promise<void> {
+	try {
+		await fs.mkdir(folder);
+	} catch (err) {
+		console.error(err);
+	}
+}
+
+/**
+ * Removes folder using fs.
```
Comment (djelinek) [inline src/ui-test/utils.ts:429]: need to modify a description after change of this method to more generic

File: src/ui-test/utils.ts [hunk 3]
```diff
@@ -395,3 +397,43 @@ export async function waitUntilEditorIsOpened(driver: WebDriver, title: string,
...
+		await fs.mkdir(folder);
+	} catch (err) {
+		console.error(err);
+	}
+}
+
+/**
+ * Removes resource using fs.
+ * 
+ * @param folder Path of resource to remove.
```
Comment (djelinek) [inline src/ui-test/utils.ts:431]: only `folder` path?

**Pull request merged**